runbotics/runbotics-orchestrator-ui/src/components/InfoPanel/ProcessInstanceEventsDetails/ProcessInstanceEventsDetailsHeader.tsx

```diff
@@ -9,12 +9,16 @@ import AccessTimeIcon from '@mui/icons-material/AccessTime';
 import {
     GridContainer, GridItem,
 } from './ProcessInstanceEventsDetails.styles';
+import {checkIfKeyExists, convertToPascalCase, translate} from '../../../hooks/useTranslations'
 
 interface Props {
     processInstanceEvent: IProcessInstanceEvent;
 }
 
-const ProcessInstanceEventsDetailsHeader: VFC<Props> = ({ processInstanceEvent }) => (
+const ProcessInstanceEventsDetailsHeader: VFC<Props> = ({ processInstanceEvent }) => {
+    const convertedKey = convertToPascalCase(processInstanceEvent.status)
+    
+    return (
```

Comment:
take a look at the comments and apply them to the whole PR, I won't duplicate them